_ml-commons-plugin/api/model-apis/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Model APIs
+parent: ML Commons API
+has_children: true
+nav_order: 10
+---
+
+# Model APIs
+
+ML Commons supports the following model-level APIs:
+
+- [Register model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/register-model/)
+- [Deploy model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/deploy-model/)
+- [Get model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/get-model/)
+- [Undeploy model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/undeploy-model/)
+- [Delete model]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/model-apis/delete-model/)
+
+## Model access control considerations
```

Comment:
This is mainly about model groups, Should we move this to `model-group-apis/index.md` ?

Reply:
This is about model groups but from the perspective of accessing a model. This section gives a brief overview of what models can be accessed by what users and then links to the model access control page for further details. So I think I'll leave this here.